.changeset/odd-buttons-pay.md

```diff
@@ -2,14 +2,21 @@
 "astro": minor
 ---
 
-Adds support for passing a metastring when using the Code component.
+Adds a new property `meta` to Astro's built-in `<Code />` component.
 
-The following code is equivalent to doing `` ```js astro=cool `` in Markdown.
+This allows you to provide a value for [Shiki's `meta` attribute](https://shiki.style/guide/transformers#meta) to pass options to transformers.
+
+The following example passes an option to highlight lines 1 and 3 to Shiki's `tranformerMetaHighlight`:
 
 ```astro
+// src/pages/index.astro
```

Comment:
This is invalid Astro markup. Comments above the `---` mark are not allowed.

Reply:
D'oh, great catch! I'm so spoiled living in the docs repo... (we always just use `title=` there)